src/main/resources/templates/polls/results.twig

```diff
@@ -26,4 +33,6 @@
 			}
 		});
 	</script>
+
+	<script src="/js/dist/vote.js"></script>
```

Comment:
This one should be `result.js`, AFAIK I already made it, if I didn't then you should copy it from vote what's necessary there (maybe some things can be removed then)